org.sasview.sasview.yml

```diff
@@ -0,0 +1,58 @@
+id: org.sasview.sasview
+runtime: org.kde.Sdk
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 6.7
+command: sasview
+finish-args:
+  - "--socket=fallback-x11"
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--device=dri"
+  - "--persist=."
+modules:
+  - name: hdf5
+    buildsystem: simple
+    build-commands:
+      - ./hdf5-1.14.5/configure --prefix=/app
+      - make -j4
+      - make install
```

Comment:
I think you can remove the versioned directory with `strip-components` https://docs.flatpak.org/en/latest/module-sources.html#archive-sources

Reply:
or just set `dest-filename: hdf5.tar.gz`

Reply:
it's an `archive` it gets extracted already strip by one level. so the command above will not work anyway because configure is indeed in the top level.

Reply:
and build would have failed if you had tried it.

Reply:
(just follow https://github.com/flathub/flathub/pull/5763#discussion_r1831054896 )

Reply:
Cheers. This makes it a lot cleaner.